cmd/dex/config.go

```diff
@@ -151,6 +151,8 @@ type Web struct {
 // Telemetry is the config format for telemetry including the HTTP server config.
 type Telemetry struct {
 	HTTP string `json:"http"`
+	// EnableProfiling makes profiling endpoints available via web interface host:port/debug/pprof/
+	EnableProfiling bool `json:"enableProfiling"`
```

Comment:
Honestly, I'd just add them, not sure if we need a switch for it, but I'm not objecting to it either.

Reply:
It was a security concern because profiling adds overhead, and frauds can use these endpoints for dos attacks.

Reply:
Ideally, the telemetry endpoint shouldn't be public, but I understand that some people publish it, so fine by me.